autorally_core/launch/autorally.launch

```diff
@@ -7,11 +7,18 @@
   <include file="$(find autorally_core)/launch/ocs.launch" />
   <include file="$(find autorally_core)/launch/autorally_core_manager.launch" />
 
+  <!-- depending on if we're using an old or new chassis, launch the appropriate interface -->
+  <group if="$(optenv AR_CHASSIS_NEW true)">
```

Comment:
Once we move to Kinetic we can evaluate the chassis name directly, but until then we'll use the additional true/false variable